infra/growth/campaigns/october.js

```diff
@@ -1,7 +1,7 @@
 const { tokenToNaturalUnits } = require('../src/util/token')
 
 
-const septemberConfig = {
```

Comment:
oops ! my bad... thanks for cleaning this up :)

Reply:
sure np :) thought it doesn't affect anything so it is more of a nitpick